app/home/home.html

```diff
@@ -29,7 +29,7 @@ <h2 class="home__title"><strong>{{ currentUser.firstName }} {{ currentUser.lastN
                     </ul>
                 </div>
                 <div class="col-xs-12 col-sm-8">
-                    <p class="h4">Звання</p>
+                    <p class="h4">{{vm.ranksNames[vm.currentUser.userRank]}}</p>
```

Comment:
Please, move logic into Controller. Create method like `vm.getUserStatus()`, which return you current user status.